aws/aws-explanation/eks-snaps.rst

```diff
@@ -0,0 +1,26 @@
+Snap Usage in EKS Worker Images
+===============================
+
+The EKS worker images bundle several snaps that provide
+kubernetes components. This page will provide a brief
+explanation on what snaps are bundled and why.
+
+Snaps
+=====
+
+.. list-table::
+   :header-rows: 1
+
+   * - **Snap Name**
+     - **Provides**
+     - **Bundled Since**
+   * - kubelet-eks
+     - kubelet binary - primary configuration mechanism
+     - EKS 1.17
+   * - kubectl-eks
+     - kubectl binary
+     - EKS 1.17
+   * - `ecr-credential-provider <https://github.com/kubernetes/cloud-provider-aws/tree/master?tab=readme-ov-file#aws-credential-provider>`_
+     - authentication mechanism for k8s with ec2 services
```

Comment:
Maybe something like:

> [AWS credential provider](https://github.com/kubernetes/cloud-provider-aws/tree/master?tab=readme-ov-file#aws-credential-provider) - an authentication mechanism for K8s that provides credentials for use with EC2 services

Are the credentials used for EC2 services in general or only for images in ECR? If they are only for ECR images, we should say that.